### PR TITLE
Update pt_br.lang

### PR DESCRIPTION
**Pull request by Mikeliro**

### CODE REVIEW

**Comment by thexaero** [comment]
Thank you for your work!
Sorry for taking so long to respond.
The translation file is not supposed to be a .json file, but the old .lang format instead. Can you please convert it with some "replaces"?

File: pt_br.lang [hunk 1]
```diff
@@ -1,255 +1,388 @@
...
+	"gui.xaero_zoom_out" : "Reduzir zoom",
+	"gui.xaero_on" : "Sim",
+	"gui.xaero_off" : "Não",
+	"gui.xaero_restores" : "Restaura",
+	"gui.xaero_preset_topleft" : "Superior esquerdo",
+	"gui.xaero_preset_topright" : "Superior direito",
+	"gui.xaero_waypoint_teleport" : "Teletransportar",
+	"gui.xaero_waypoints_key" : "Waypoints",
+	"gui.xaero_display_coords" : "Exibir coordenadas",
+	"gui.xaero_disable_enable" : "Ativar/Desativar",
```
Comment (Greg-21) [inline pt_br.lang:123]: The order shouldn't be changed and should match the following pattern: `<Text_disable_here>`/`<Text_enable_here>`.

It's also a quick reminder for any future reviewer (most likely for me).

Comment (Greg-21) [inline pt_br.lang:123]: The order shouldn't be changed here and should match the following pattern: `<Text_disable_here>`/`<Text_enable_here>`.

It's also a quick reminder for any future reviewer (most likely for me).

Comment (Greg-21) [inline pt_br.lang:123]: The order here shouldn't be changed and should match the following pattern: `<Text_disable_here>`/`<Text_enable_here>`.

It's also a quick reminder for any future reviewer (most likely for me).

**Comment by thexaero** [comment]
The author seems to have disappeared, so I'm closing this.

**Comment by Greg-21** [comment]
They are still active on GitHub, but not here for some reason.

**Comment by thexaero** [comment]
Well, I take it as them not being interested anymore, and that's fine.

**Pull request closed without merging**